lib/testing_api_onlyoffice_com/test_instance/main_page/community_server_api.rb

```diff
@@ -1,200 +1,67 @@
+require 'json'
+require 'onlyoffice_file_helper'
+require_relative '../../../../lib/testing_api_onlyoffice_com/test_instance/main_page/helper_for_api_documentation/document_entry'
```

Comment:
Also required in https://github.com/onlyoffice-testing-robot/testing-api.onlyoffice.com/blob/76be964253c9f233ad65adee0b1e615c356404eb/lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api/document_builder_api_common/builder_page.rb#L3
One place is enough